arangosh/Utils/ClientManager.cpp

```diff
@@ -80,7 +80,7 @@ ClientManager::~ClientManager() {}
 
 Result ClientManager::getConnectedClient(
     std::unique_ptr<httpclient::SimpleHttpClient>& httpClient, bool force,
-    bool verbose) {
+    bool logServerVersion, bool logDatabaseNotFound) {
```

Comment:
Good. I never really did like the "verbose" flag given that it really only enabled one log statement.